boards/arm/nrf52dk_nrf52832/nrf52dk_nrf52832.dts

```diff
@@ -200,6 +200,10 @@ arduino_spi: &spi2 {
 	pinctrl-names = "default", "sleep";
 };
 
+&nfct {
+	status = "okay";
```

Comment:
Setting OK at at soc level is not ok. You can use the SOC without NFC peripheral utilized externally. It's ok to set at board level, because these boards have NFC used.